config.py

```diff
@@ -0,0 +1,7 @@
+import os
+from dotenv import load_dotenv
```

Comment:
Uma das alternativas é utilizar o python-decouple.

```suggestion
from decouple import config as ENV
```